### PR TITLE
Fix translate english to korean in message properties file

### PR DESCRIPTION
**Pull request by chd830**
## Description
Translate msg.common.*

## Type of change
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Documentation update
- [ ] Refactoring, Maintenance
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### CODE REVIEW

File: src/main/resources/messages/message_ko_KR.properties [hunk 1]
```diff
@@ -1,24 +1,23 @@
...
-error.common.system=An error occurred. Please Retry or Contact the system administrator. <br /> {0}
-msg.common.success=Successfully completed.
-msg.common.report.select.sheet=Please Select more than one sheet
-msg.common.confirm.delete=Do you want to delete selected record(s)?
-msg.common.confirm.reset=Do you want to clear all record(s)?
-msg.common.confirm.save=Do you want to save?
-msg.common.confirm.continue=Do you want to continue?
-msg.common.confirm.mergeAndSave=Identification was changed. Do you want to merge and save?
-msg.common.search.check.date=Please check the period again.
+login.lan.ko=한국어
```
Comment (suhwan-cheon) [inline src/main/resources/messages/message_ko_KR.properties:3]: comment

Comment (suhwan-cheon) [inline src/main/resources/messages/message_ko_KR.properties]:
```suggestion
login.lan.ko=korean
```

Comment (chd830) [inline src/main/resources/messages/message_ko_KR.properties]:
```suggestion
login.lan.ko=korean
```

File: src/main/resources/messages/message_ko_KR.properties [hunk 2]
```diff
@@ -1,24 +1,23 @@
...
-error.common.system=An error occurred. Please Retry or Contact the system administrator. <br /> {0}
-msg.common.success=Successfully completed.
-msg.common.report.select.sheet=Please Select more than one sheet
-msg.common.confirm.delete=Do you want to delete selected record(s)?
-msg.common.confirm.reset=Do you want to clear all record(s)?
-msg.common.confirm.save=Do you want to save?
-msg.common.confirm.continue=Do you want to continue?
-msg.common.confirm.mergeAndSave=Identification was changed. Do you want to merge and save?
-msg.common.search.check.date=Please check the period again.
+login.lan.ko=korean
```
Comment (chd830) [inline src/main/resources/messages/message_ko_KR.properties]:
```suggestion
lan.ko=korean
```

File: src/main/resources/messages/message_ko_KR.properties [hunk 3]
```diff
@@ -1,24 +1,23 @@
...
+login.lan.en=영어
```
Comment (suhwan-cheon) [inline src/main/resources/messages/message_ko_KR.properties]:
```suggestion
lan.en=english
```

File: src/main/resources/messages/message_ko_KR.properties [hunk 4]
```diff
@@ -2,23 +2,22 @@
...
-msg.common.search.check.date=Please check the period again.
+
+msg.login.failed=잘못된 아이디 또는 패스워드입니다. 다시 시도해주세요.
+
+msg.common.window.allowpopup=브라우저의 팝업을 허용해주세요.
+
+msg.common.valid=입력칸을 모두 채우고 다시 시도해주세요.
+msg.common.valid2=에러가 발생했습니다. 다시 시도하거나 시스템 관리자에게 연결하세요.
+error.common.system=에러가 발생했습니다. 다시 시도하거나 시스템 관리자에게 연결하세요.
+msg.common.success=성공적으로 완료되었습니다.
```
Comment (suhwan-cheon) [inline src/main/resources/messages/message_ko_KR.properties:13]:
```suggestion
msg.common.success=성공적으로 완료되었습니다. <br /> {0}
```

File: src/main/resources/messages/message_ko_KR.properties [hunk 5]
```diff
@@ -2,23 +2,22 @@
...
-msg.common.search.check.date=Please check the period again.
+
+msg.login.failed=잘못된 아이디 또는 패스워드입니다. 다시 시도해주세요.
+
+msg.common.window.allowpopup=브라우저의 팝업을 허용해주세요.
+
+msg.common.valid=입력칸을 모두 채우고 다시 시도해주세요.
+msg.common.valid2=에러가 발생했습니다. 다시 시도하거나 시스템 관리자에게 연결하세요.
+error.common.system=에러가 발생했습니다. 다시 시도하거나 시스템 관리자에게 연결하세요.
+msg.common.success=성공적으로 완료되었습니다. <br /> {0}
```
Comment (suhwan-cheon) [inline src/main/resources/messages/message_ko_KR.properties]:
```suggestion
msg.common.success=성공적으로 완료되었습니다.
```

File: src/main/resources/messages/message_ko_KR.properties [hunk 6]
```diff
@@ -2,23 +2,22 @@
...
-msg.common.confirm.mergeAndSave=Identification was changed. Do you want to merge and save?
-msg.common.search.check.date=Please check the period again.
+
+msg.login.failed=잘못된 아이디 또는 패스워드입니다. 다시 시도해주세요.
+
+msg.common.window.allowpopup=브라우저의 팝업을 허용해주세요.
+
+msg.common.valid=입력칸을 모두 채우고 다시 시도해주세요.
+msg.common.valid2=에러가 발생했습니다. 다시 시도하거나 시스템 관리자에게 연결하세요.
+error.common.system=에러가 발생했습니다. 다시 시도하거나 시스템 관리자에게 연결하세요.
```
Comment (suhwan-cheon) [inline src/main/resources/messages/message_ko_KR.properties]:
```suggestion
error.common.system=에러가 발생했습니다. 다시 시도하거나 시스템 관리자에게 연결하세요. <br /> {0}
```

File: src/main/resources/messages/message_ko_KR.properties [hunk 7]
```diff
@@ -2,23 +2,22 @@
...
-error.common.system=An error occurred. Please Retry or Contact the system administrator. <br /> {0}
-msg.common.success=Successfully completed.
-msg.common.report.select.sheet=Please Select more than one sheet
-msg.common.confirm.delete=Do you want to delete selected record(s)?
-msg.common.confirm.reset=Do you want to clear all record(s)?
-msg.common.confirm.save=Do you want to save?
-msg.common.confirm.continue=Do you want to continue?
-msg.common.confirm.mergeAndSave=Identification was changed. Do you want to merge and save?
-msg.common.search.check.date=Please check the period again.
+
```
Comment (suhwan-cheon) [inline src/main/resources/messages/message_ko_KR.properties]:
```suggestion
lan.default=-select language-
```

**Pull request merged**